utils/json/nullable/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package nullable provides nullable types that get serialized to JSON null.
```

Comment:
This file should be `nullable.go`.

Reply:
Done. Also did `option/doc.go` -> `option/option.go`